inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -155,6 +156,25 @@ def _try_to_log(logfn, *args, **kwargs):
     return _decorator
 
 
+def ignore_timeout_error(func):
+    """Ignore the TimeoutError.
+
+    Quick fix for ``refextract`` and ``plotextract`` tasks only. It
+    shouldn't be used for others!
+    """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except TimeoutError:
+            LOGGER.debug(
```

Comment:
I would log this with higher priority so it makes it to sentry, and also put the `func.__name__` in the message. If you use the `stackTraceLogger` from `inspire-utils`, you don't have to pass the traceback manually, as it will be handled automatically by raven.